pkg/test/resources/samples/meshcontext_samples.go

```diff
@@ -0,0 +1,36 @@
+package samples
+
+import (
+	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	test_model "github.com/kumahq/kuma/pkg/test/resources/model"
+	test_xds "github.com/kumahq/kuma/pkg/test/xds"
+	xds_context "github.com/kumahq/kuma/pkg/xds/context"
+)
+
+func CreateSimpleMeshContext() xds_context.Context {
+	return CreateSimpleMeshContextWith(xds_context.NewResources())
+}
+
+func CreateSimpleMeshContextWith(resources xds_context.Resources) xds_context.Context {
```

Comment:
I'd change Simple to Sample (same with other func). For me, simple suggest minimal working object, but this one have sample data.